logic-apps/pull-reference-data.json

```diff
@@ -619,6 +690,101 @@
         },
         "type": "If"
       },
+      "IfActiveAssetUnivariateAnomalyDetectionScenarioExists": {
+        "actions": {
+          "GetAssetUnivariateAnomalyDetectionParams": {
+            "inputs": {
+              "authentication": "@parameters('DynamicsIdentityAuthentication')",
+              "method": "GET",
+              "uri": "@{parameters('EnvironmentUrl')}data/AnomalyDetectionParams"
+            },
+            "runAfter": {},
+            "type": "Http"
+          },
+          "ParseAssetUnivariateAnomalyDetectionParams": {
+            "inputs": {
+              "content": "@body('GetAssetUnivariateAnomalyDetectionParams')",
+              "schema": {
+                "properties": {
+                  "value": {
+                    "items": {
+                      "properties": {
```

Comment:
As I understand from the code we dont really need `IsSensorActiveForScenario` and `Scenario` as it is handled in some other if/else statements. Other scenarios dont use it 🤔

Reply:
correct